ddl/job_table_test.go

```diff
@@ -184,3 +190,87 @@ func check(t *testing.T, record []int64, ids ...int64) {
 		}
 	}
 }
+
+func TestConcurrentDDLSwitch(t *testing.T) {
```

Comment:
It's better to check some states when switching.

Reply:
checked